src/window.rs

```diff
@@ -1396,6 +1396,18 @@ impl Window {
             .maybe_wait_on_main(|w| w.drag_resize_window(direction))
     }
 
+    /// Show system menu on current mouse position.
+    ///
+    /// There's no guarantee that calling this method show the system menu.
+    /// user can cancel opening of it before it even open.
```

Comment:
If I understood you correctly, it might be good for users to know that this action only works properly if connected to a mouse click, similar to [`Window::drag_window()`](https://docs.rs/winit/0.28.6/winit/window/struct.Window.html#method.drag_window).

Reply:
No, this isn't necessary.

I remember it was necessary in `Window::drag_window()` because it was required by the API in some backends. If it's not required for `Window::show_system_menu()` we shouldn't add any unnecessary constraints.

Reply:
> one thing that we can do is allow passing the position of menu to the `show_system_menu` so user can choose where the menu should open... it can be a Option, so if None we use mouse position

It doesn't sound that important to me, but I have no objections if you want to add that.

Reply:
Personally I'm happy to merge like this, but if you want to add more text please feel free to do so!

Maybe we also want to explain what the "system menu" is actually, or maybe just add a link to [Wikipedia](https://en.wikipedia.org/wiki/Common_menus_in_Microsoft_Windows#System_menu). Again, up to you really.

Reply:
That would be much appreciated.